x-pack/plugins/cases/public/components/configure_cases/connectors.tsx

```diff
@@ -95,13 +96,13 @@ const ConnectorsComponent: React.FC<Props> = ({
       >
         <EuiFormRow
           fullWidth
-          css={css`
-            label {
-              width: 100%;
-            }
-          `}
           label={dropDownLabel}
           data-test-subj="case-connectors-form-row"
+          labelAppend={
```

Comment:
I think we should hide the label if the user does not have access to create connectors. You can check this by doing

```
const { actions } = useApplicationCapabilities();
const canSave = actions.crud;

labelAppend={canSave ? <EuiButtonEmpty // rest of code /> : undefined}
```

Reply:
done, thanks!